### PR TITLE
Update parity-wasm to 0.37.0

### PR DESCRIPTION
**Pull request by lzutao**
This is an alternative to #85.

Closes #85.

### CODE REVIEW

**Comment by lzutao** [comment]
@philipc Could you transition Travis build to `travis-ci.com`?

**Comment by philipc** [comment]
> Could you transition Travis build to travis-ci.com?

What's the benefit? From a quick look, that's not automated and [still beta](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/#existing-open-source-repositories-on-travis-ciorg)?

File: .travis.yml [hunk 1]
```diff
@@ -1,37 +1,39 @@
-sudo: required
```
Comment (philipc) [inline .travis.yml]: I think this is required for tarpaulin. Or at least it would be if we had any tests to run.

**Comment by philipc** [comment]
Please put travis changes in a separate PR.

File: .travis.yml (hunk 1 above)
Comment (lzutao) [reply .travis.yml]: sudo is [no-effect now](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration).

**Comment by lzutao** [comment]
I opened travis PR in #93. The benefit I could tell is:
- travis-ci.com use GitHub Apps Integration, which means Travis' GitHub App requires far [less permissions](https://docs.travis-ci.com/user/migrate/open-source-on-travis-ci-com/#features-of-the-github-apps-integration) than our OAuth application.
- now we only need one domain to control build service.

You could know more in https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps.

File: src/wasm.rs [hunk 2]
```diff
@@ -211,6 +211,7 @@ impl<'file> ObjectSection<'static> for WasmSection<'file> {
             elements::Section::Start(_) => Some("Start"),
             elements::Section::Element(_) => Some("Element"),
             elements::Section::Code(_) => Some("Code"),
+            elements::Section::DataCount(_) => Some("Data"),
```
Comment (philipc) [inline src/wasm.rs]:
```suggestion
            elements::Section::DataCount(_) => Some("DataCount"),
```

File: src/wasm.rs [hunk 3]
```diff
@@ -236,6 +237,7 @@ impl<'file> ObjectSection<'static> for WasmSection<'file> {
             elements::Section::Start(_) => SectionKind::Other,
             elements::Section::Element(_) => SectionKind::Other,
             elements::Section::Code(_) => SectionKind::Text,
+            elements::Section::DataCount(_) => SectionKind::Data,
```
Comment (philipc) [inline src/wasm.rs]:
```suggestion
            elements::Section::DataCount(_) => SectionKind::Other,
```

**Comment by philipc** [comment]
I see no urgent need to switch to travis-ci.com, and since it is still beta I don't want to spend time on it now.

**Pull request merged**